python/ray/util/serialization.py

```diff
@@ -16,3 +16,14 @@ def register_serializer(cls, *, serializer, deserializer):
     """
     context = ray.worker.global_worker.get_serialization_context()
     context._register_cloudpickle_serializer(cls, serializer, deserializer)
+
+
+def unregister_serializer(cls):
```

Comment:
Let's rename this to `deregister_serializer`, it is a bit more common usage in this circumstance I think :)